src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj

```diff
@@ -127,7 +127,7 @@
             We have to run restore first with the PublishReadyToRun flag set to true to ensure that the correct crossgen packages get restored.
             See https://github.com/dotnet/sdk/issues/20701
         -->
-        <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="Restore" Properties="RestoreWithR2R=true" />
+        <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="Restore" Properties="RestoreWithR2R=true;RestoreUseStaticGraphEvaluation=false" />
```

Comment:
Should we file a tracking issue to revert this?

Reply:
If nothing else, we should have a comment explaining it.